contracts/VestedPayment.sol

```diff
@@ -0,0 +1,81 @@
+pragma solidity ^0.4.13;
+
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+import "./LifToken.sol";
+
+contract VestedPayment is Ownable {
+  using SafeMath for uint256;
+
+  // when the vested schedule starts
+  uint256 public startTime;
+
+  // how much seconds each period will last
+  uint256 public secondsPerPeriod;
+
+  // how much periods will have in total
+  uint256 public totalPeriods;
+
+  // the amount of tokens to be vested in total
+  uint256 public tokens;
+
+  // how much tokens were claimed
+  uint256 public claimed;
+
+  // the token contract
+  LifToken public token;
+
+  // duration (in periods) of the initial cliff in the vesting schedule
+  uint256 public cliffDuration;
+
+  // if the vested payment was funded or not
+  bool public funded = false;
+
+  function VestedPayment(
+    uint256 _startTime, uint256 _secondsPerPeriod,
+    uint256 _totalPeriods, uint256 _cliffDuration, address tokenAddress
```

Comment:
should we `require(cliffDuration < totalPeriods)`? It avoids any confusion if somebody thinks the duration units is seconds when it's periods. I know it's explained in the `cliffDuration` comment but...

Reply:
yes, we should. adding it.